sympy/series/limits.py

```diff
@@ -212,6 +212,20 @@ def doit(self, **hints):
 
         e, z, z0, dir = self.args
 
+        if str(dir) == '+-':
```

Comment:
Is `str` necessary?

Reply:
Yes , `dir` from if of type `Symbol`  after getting it through `e, z, z0, dir = self.args` , so a type conversion is needed here . Actually if you have a look in `limits.py` , most of the times you'll find `str(dir)` being used for this reason. Resolving !